yt_dlp/extractor/bbc.py

```diff
@@ -1255,6 +1266,45 @@ def extract_all(pattern):
                 lambda s: self._parse_json(s, playlist_id, fatal=False),
                 re.findall(pattern, webpage))))
 
+        # US accessed article with single embedded video (e.g.
+        # https://www.bbc.com/news/uk-68546268)
+        video_id = self._match_id(url)
+        next_data = self._search_nextjs_data(webpage, video_id)['props']['pageProps']['page']
```

Comment:
this needs to be non-fatal -- use `traverse_obj`. as-is, this line will `KeyError` for UK users before the rest of the extraction code can be reached